include/common_structs.h

```diff
@@ -411,12 +411,12 @@ struct Entity;
 
 // BEGIN ENTITY-SPECIFIC STRUCTS
```

Comment:
Wondering if it would be wise to move this section to `entity.h`

Reply:
all these structs are not referenced anywhere outside entity folder so it might be good idea